tests/test_app/test_align.py

```diff
@@ -634,3 +639,48 @@ def test_progressive_align_iters(seqs):
     assert isinstance(aln, ArrayAlignment)
     assert len(aln) == 42
     assert aln.moltype == aligner._moltype
+
+
+def test_smith_waterman_matches_local_pairwise(seqs):
+    aligner = smith_waterman()
+    got = aligner(seqs.get_seq("Human"), seqs.get_seq("Bandicoot"))
```

Comment:
change test to using a sequence collection. Add a new test for calling with a sequence collection that does not have 2 sequences.